python/hyperonpy.cpp

```diff
@@ -5,14 +5,15 @@
 
 namespace py = pybind11;
 
-template<class T, size_t N>
+template <class T, size_t N>
```

Comment:
Please remove space and code formatting changes as they make reviewing code difficult.